pkg/api/operators/apis/experiment/v1alpha2/experiment_types.go

```diff
@@ -50,6 +50,9 @@ type ExperimentSpec struct {
 	// TODO - figure out what to do with metric collectors
 	MetricsCollectorType string `json:"metricsCollectorSpec,omitempty"`
```

Comment:
Is this needed?

Reply:
I think we can keep it here, since MetricsCollectorSpec is a workaround. In long term, we will use MetricsCollectorType

Reply:
If we need to change it in the future, should we deprecate metricsCollectorSpec directly or create a new API version?

Reply:
I suggest just  deprecate metricsCollectorSpec if there is no big change

Reply:
I feel that we needn't include any fields that are not supported in current release.  We will need a new API version anyways.